shared/src/business/entities/User.test.js

```diff
@@ -98,12 +98,34 @@ describe('User entity', () => {
     expect(user.isValid()).toBeTruthy();
   });
 
+  it('Creates a valid active judge user', () => {
+    const user = new User({
+      judgeFullName: 'Saul Perfectly Goodman',
+      judgeTitle: 'Chief Judge',
+      name: 'Saul Goodman',
+      role: ROLES.judge,
+      userId: '5488fed1-1129-4ca0-be7a-3ea3998be953',
+    });
+    expect(user.isValid()).toBeTruthy();
+  });
+
+  it('Creates a valid active judge user', () => {
```

Comment:
I think this should say `Creates a valid legacy judge user`.

Reply:
Good catch! Fix incoming.

Reply:
Fixed in https://github.com/ustaxcourt/ef-cms/pull/455/commits/5d72eeb90b0e26af5125fec353cf1ca36883a1b6